### DISCUSSION TITLE
Link to active file

### DISCUSSION THREAD

**Discussion by d80harri** [SilentVoid13/Templater#949] [Help]
Hey everybody.

I want to implement an incremental reading workflow. The manual steps I do currently are

1. I open a note and start reading from top
2. While doing so I delete everything that is unimportant
3. If I find something important I cut it, create a new note, add a link to the note that I was just reading, paste the content, and rewrite the content

Not sure how templater can help me here. For now I just tried to create a template that links to the page that was open when I created the new note. I tried ` <% tp.config.active_file.name %>`as well as `<% tp.config.target_file.name %>` but both seem to be the same file (the newly created file)

So my questions: Whats the difference between active_file and target_file? 
and
Is there away to insert this link back to the original file?

**Comment by Zachatoo** [comment]
Have you tried the [note refactor](https://github.com/lynchjames/note-refactor-obsidian) plugin? Or the built in plugin for refactoring notes? They may better suit your use case.
